ark/utils/test_utils.py

```diff
@@ -580,6 +581,52 @@ def create_test_extraction_data():
     return cell_mask, channel_data
 
 
+def save_figure(save_dir, save_file):
+    """Verify save_dir and save_file, then save to specified location
+
+    Args:
+        save_dir (str):
+            the name of the directory we wish to save to
+        save_file (str):
+            the name of the file we wish to save to
+    """
+
+    # verify save_dir exists
+    if not os.path.exists(save_dir):
+        raise FileNotFoundError("save_dir %s does not exist" % save_dir)
+
+    # verify that if save_dir specified, save_file must also be specified
+    if save_file is None:
+        raise FileNotFoundError("save_dir specified but no save_file specified")
+
+    plt.savefig(os.path.join(save_dir, save_file))
+
+
+def verify_in_list(list_verify, values_bucket, name_list_verify, name_values_bucket):
```

Comment:
I think changing the variables to something like `test_list` and `good_values` would make this more clear. I also think it would be helpful to call these arguments by name when using the function. Otherwise, it's fairly hard to figure out what's going on when someone is reading through the code and just sees 4 different lists/names being passed to a verify function